Tools/setup/ubuntu.sh

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env bash
+
+# check ubuntu version
+# instructions for 16.04, 18.04
+# otherwise warn and point to docker?
+UBUNTU_RELEASE=`lsb_release -rs`
+
+if [[ "${UBUNTU_RELEASE}" == "14.04" ]]
+then
+	echo "Ubuntu 14.04 unsupported, see docker px4io/px4-dev-base"
+	exit 1
+elif [[ "${UBUNTU_RELEASE}" == "16.04" ]]
+then
+	echo "Ubuntu 16.04"
+elif [[ "${UBUNTU_RELEASE}" == "18.04" ]]
+then
+	echo "Ubuntu 18.04"
+	echo "WARNING, instructions only tested on Ubuntu 16.04"
+fi
+
+export DEBIAN_FRONTEND=noninteractive
+sudo apt-get update --quiet
+sudo apt-get -y --quiet --no-install-recommends install \
```

Comment:
- This is a bigger subset that the original scripts - do we need/should we have everything? 
- Is there anything here that might "conflict" = e.g. is n't gcc going to have compiler tools that might not be the version we need?